Don't use -Werror
That causes newer clang compilers to fail with:
error: lambda capture 'silent' is not used [-Werror,-Wunused-lambda-capture]

The target "doc/example-config" is a dependency of "install-examples" only, so
it runs in the destroot phase, add it to "all" to avoid that. 
--- Makefile
+++ Makefile
@@ -18,7 +18,7 @@
 DEFINES+=-DGIT_HASH=\"$(GIT_HASH)\"
 endif
 
-WARNFLAGS=-Werror -Wall -Wextra -Wunreachable-code
+WARNFLAGS=-Wall -Wextra -Wunreachable-code
 INCLUDES=-Iinclude -Istfl -Ifilter -I. -Irss
 BARE_CXXFLAGS=-std=c++11 -O2 -ggdb $(INCLUDES)
 LDFLAGS+=-L.
@@ -81,7 +81,7 @@
 TEXTCONV=./txt2h
 RM=rm -f
 
-all: doc $(NEWSBOAT) $(PODBOAT) mo-files
+all: doc $(NEWSBOAT) $(PODBOAT) mo-files doc/example-config
 
 NB_DEPS=xlicense.h $(LIB_OUTPUT) $(FILTERLIB_OUTPUT) $(NEWSBOAT_OBJS) $(RSSPPLIB_OUTPUT)
 
